app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,104 @@
+
+    <string name="open_intent_browser">Öffne Browser…</string>
+    <string name="open_intent_browser_error">Fehler beim Öffnen der Details, bitte URL checken</string>
+
+    <string name="action_website">Webseitenaktion</string>
+
+    <string name="dialog_website_new_title">Neue Webseite</string>
+    <string name="dialog_website_edit_title">Editieren</string>
+    <string name="dialog_website_add">Hinzufügen</string>
+    <string name="dialog_website_error_itemperpage">Die Anzahl der Objekte pro Webseite müssen mehr als 0 sein</string>
+    <string name="dialog_website_error_name">Möchtest du nicht einen Namen hinzufügen?</string>
+    <string name="dialog_website_error_pagename">Füge einen Namen hinzu</string>
+    <string name="dialog_website_error_url">Fehlende URL :(</string>
+    <string name="dialog_website_error_selector">Missing text selector</string>
+    <string name="dialog_website_save">Speichern</string>
+    <string name="dialog_website_name_hint">Webseiten Name</string>
+    <string name="dialog_website_page_name_hint">Seitenname</string>
+    <string name="dialog_website_url_hint">Webseiten URL</string>
+    <string name="dialog_website_urlsuffix_hint">(optional) Page number suffix</string>
+    <string name="dialog_website_selector_hint">DOM Selektor</string>
+    <string name="dialog_website_itemperpage_hint">Anzahl der Objekte auf einer Seite</string>
+    <string name="dialog_website_firstpagezero_hint">Wenn die Nummer der ersten Seite 0 ist</string>
+
+    <string name="dialog_websitechooser_title">Wähle deine Favoriten</string>
+
+
+    <!-- About fragment -->
+    <string name="about_title">Über</string>
+    <string name="about_made">Gemacht mit Liebe von</string>
+    <string name="about_made_person" translatable="false">Hugo Gresse http://hugo.gresse.io/</string>
+    <string name="using" translatable="false">Third-party libraries</string>
+    <string name="error_server_unknown" translatable="false">Something went wrong with the server :(</string>
+    <string-array name="about_libraries" translatable="false">
+        <item>OkHttp by Square http://square.github.io/okhttp/</item>
+        <item>Otto by Square http://square.github.io/otto/</item>
+        <item>Gson by Google http://github.com/google/gson</item>
+        <item>Jsoup by Jonathan Hedley http://jsoup.org/</item>
+        <item>ButterKnife by Jake Wharton http://jakewharton.github.io/butterknife/</item>
+        <item>Glide by Bumptech http://github.com/bumptech/glide</item>
+        <item>PhotoView by Chris Banes http://github.com/chrisbanes/PhotoView</item>
+        <item>SimpleExoPlayer by Hugo Gresse http://github.com/HugoGresse/SimpleExoPlayer</item>
+    </string-array>
+
+    <!-- Strings related to Settings -->
+    <string name="pref_cat_display">Display</string>
+    <string name="pref_rowstriping_title">Quote row striping</string>
+    <string name="pref_rowstriping_summary">Add a background to separate each quotes</string>
```

Comment:
enable row stripping change the background color for each item, to be mroe readable. (aka %2)